src/random_number_generator.h

```diff
@@ -0,0 +1,33 @@
+#ifndef CYCLUS_SRC_RNG_H
+#define CYCLUS_SRC_RNG_H
+
+#include <boost/random.hpp>
+
+class SimInitTest;
+
+namespace cyclus {
+
+class Context;
+class SimInfo;
+
+/// A random number generator.
+class RandomNumberGenerator {
+    friend class ::SimInitTest;
+  public:
+    RandomNumberGenerator();
+
+    ~RandomNumberGenerator();
+
+    /// Initialize from seed
+    void Initialize(Context* ctx, SimInfo si);
```

Comment:
Do we need the `Context`? We don't save it/use it. There may be long term value in storing the `Context` in this class, but I don't see a reason right now.